internal/resource/config/captchaproviders/captcha_providers_resource_gen.go

```diff
@@ -0,0 +1,301 @@
+// Code generated by ping-terraform-plugin-framework-generator
+
+package captchaproviders
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	client "github.com/pingidentity/pingfederate-go-client/v1210/configurationapi"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/resource/common/pluginconfiguration"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/resource/config"
+	internaltypes "github.com/pingidentity/terraform-provider-pingfederate/internal/types"
+)
+
+var (
+	_ resource.Resource                = &captchaProvidersResource{}
+	_ resource.ResourceWithConfigure   = &captchaProvidersResource{}
+	_ resource.ResourceWithImportState = &captchaProvidersResource{}
+)
+
+func CaptchaProvidersResource() resource.Resource {
+	return &captchaProvidersResource{}
+}
+
+type captchaProvidersResource struct {
+	providerConfig internaltypes.ProviderConfiguration
+	apiClient      *client.APIClient
+}
+
+func (r *captchaProvidersResource) Metadata(_ context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_captcha_providers"
```

Comment:
This resource name should be singular - you'll need to regenerate this file

Reply:
done